bridge/evm/test/CommitteeUpgradeableTest.t.sol

```diff
@@ -204,6 +198,30 @@ contract CommitteeUpgradeableTest is BridgeBaseTest {
         bridge.upgradeWithSignatures(signatures, message);
     }
 
+    function testInitializeImplementationDisabled() public {
+        // Deploy the implementation contract without proxy
+        address implementation = address(new BridgeCommittee());
+
+        // Prepare initialization data
+        address[] memory _committeeMembers = new address[](5);
+        _committeeMembers[0] = committeeMemberA;
+        _committeeMembers[1] = committeeMemberB;
+        _committeeMembers[2] = committeeMemberC;
+        _committeeMembers[3] = committeeMemberD;
+        _committeeMembers[4] = committeeMemberE;
+        uint16[] memory _stake = new uint16[](5);
+        _stake[0] = 1000;
+        _stake[1] = 1000;
+        _stake[2] = 1000;
+        _stake[3] = 2002;
+        _stake[4] = 4998;
+
+        // Try to call the initializer directly on the implementation contract
+        // Expect it to revert because initializers should be disabled
+        vm.expectRevert();
+        BridgeCommittee(implementation).initialize(_committeeMembers, _stake, 100);
```

Comment:
do we want to assert the same for every CommitteeUpgradable impl contract?

Reply:
they all reference the same `_disableInitializers()` call in the base `CommitteeUpgradeable` contract so if it works for one it should work for all.